src/edu/csus/ecs/pc2/ui/SubmitSampleRunsPane.java

```diff
@@ -0,0 +1,877 @@
+		selectedNosLabel.addMouseListener(new MouseAdapter() {
+			@Override
+			public void mouseClicked(MouseEvent e) {
+			    showJudgementTypeList();
+			}
+		});
+		submitNoSamplesPane.add(selectedNosLabel);
+		JButton selectJudementTypesButton = new JButton("Select");
+		selectJudementTypesButton.setToolTipText("Select Judgement Types");
+		selectJudementTypesButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+			    showJudgementTypeList();
+			}
+		});
+		
+		submitNoSamplesPane.add(selectJudementTypesButton);
+		
+		JLabel lblNewLabel_2 = getWhatsThisOne();
+		submitNoSamplesPane.add(lblNewLabel_2);
+		
+		JPanel submitNoSamplesPane_1 = new JPanel();
+		FlowLayout flowLayout_2 = (FlowLayout) submitNoSamplesPane_1.getLayout();
+		flowLayout_2.setAlignment(FlowLayout.LEFT);
+		submitNoSamplesPane_1.setBorder(new TitledBorder(null, "", TitledBorder.LEADING, TitledBorder.TOP, null, null));
+		submitNoSamplesPane_1.setBounds(33, 161, 650, 32);
+		controlsPane.add(submitNoSamplesPane_1);
+		   
+        checkBoxProblems = new JCheckBox("Problems");
+        checkBoxProblems.setVerticalAlignment(SwingConstants.TOP);
+        checkBoxProblems.setToolTipText("Select Problems");
+        checkBoxProblems.setSelected(true);
+        submitNoSamplesPane_1.add(checkBoxProblems);
+        
+        submitAllProblemsRadioButton = new JRadioButton("Submit All");
+        submitAllProblemsRadioButton.setSelected(true);
+        submitNoSamplesPane_1.add(submitAllProblemsRadioButton);
+        
+        submitSelectedProblemsRadioButton = new JRadioButton("Submit Selected");
+        submitNoSamplesPane_1.add(submitSelectedProblemsRadioButton);
+        
+        selectedProblemsLabel = new JLabel("None selected");
+        selectedProblemsLabel.setToolTipText("None selected");
+        submitNoSamplesPane_1.add(selectedProblemsLabel);
+        
+        selectProblemsButton = new JButton("Select");
+        selectProblemsButton.addActionListener(new ActionListener() {
+            public void actionPerformed(ActionEvent e) {
+                showProblemList();
+            }
+        });
+        selectProblemsButton.setToolTipText("Select Problems");
+        submitNoSamplesPane_1.add(selectProblemsButton);
+        
+        JPanel submitLanguages = new JPanel();
+        FlowLayout fl_submitLanguages = (FlowLayout) submitLanguages.getLayout();
+        fl_submitLanguages.setAlignment(FlowLayout.LEFT);
+        submitLanguages.setBorder(new TitledBorder(null, "", TitledBorder.LEADING, TitledBorder.TOP, null, null));
+        submitLanguages.setBounds(33, 191, 650, 32);
+        controlsPane.add(submitLanguages);
+        
+        checkBoxLanguage = new JCheckBox("Languages");
+        checkBoxLanguage.setVerticalAlignment(SwingConstants.TOP);
+        checkBoxLanguage.setToolTipText("Select Languages");
+        checkBoxLanguage.setSelected(true);
+        submitLanguages.add(checkBoxLanguage);
```

Comment:
This is an example of what I was talking about (above):  it would help the reader/reviewer/maintainer if this statement were instead `submitLanguagesPane.add(checkBoxLanguage)`.